.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,7 +12,7 @@ assignees: ''
 
 ### Checklist
 
-- [] Verified whether your issue exists in the latest version Mux Element(s) release: `npm install @mux-elements/[element_name]` or `yarn @mux-elements/[element_name]`.
+- [] Verified whether your issue exists in the latest version Mux Element(s) release: `npm install @mux/[element_name]` or `yarn @mux/[element_name]`.
```

Comment:
without the codemod change below, only the first instance of `@mux-elements` below would've gotten updated.